README.md

```diff
@@ -1,6 +1,6 @@
 # Shed
 
-A collection of tools for Workers
+> A collection of tools for Workers
```

Comment:
How about:

A collection of tools for [service workers](https://slightlyoff.github.io/ServiceWorker/spec/service_worker/)